server/src/test/java/org/apache/iotdb/db/integration/IoTDBRemovePartitionIT.java

```diff
@@ -174,7 +174,7 @@ private static void insertData() throws ClassNotFoundException {
         sqls.add(String.format("INSERT INTO root.test%d(timestamp, s0) VALUES (%d, %d)", j,
             i * partitionInterval, i * partitionInterval));
       }
-      sqls.add("MERGE");
+      // sqls.add("MERGE");
```

Comment:
Does this merge cause any trouble?

Reply:
Oops, I forget to fix this after debugging~ I will fix now